### PR TITLE
Small Node Popover Menu Refactor

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
Was looking into another task related to the kebab-menu for scene graph nodes in the scene menu and came across some redundant code and issues with constant rerenders of panels due to `useSelector` returning new object references every render call. This PR collects those small fixes.

The code for `focusNodePropertiesPanel` only added a "Current Focus" string to the title of the Properties panel window. Considering how much code was used for this, I found it redundant and removed everything related to this feature. Made the code a bit cleaner

### CODE REVIEW

File: src/components/NodeMetaPanel/NodeMetaContainer.jsx [hunk 1]
```diff
@@ -6,10 +6,8 @@ import NodeMetaPanel from './NodeMetaPanel';
 import styles from '../NodePropertiesPanel/NodePopOverContainer.scss';
 
 function NodeMetaContainer() {
-  const activeNodeMetaPanels = useSelector((state) => {
-    const panels = state.local.popovers.activeNodeMetaPanels;
-    return [...Object.keys(panels)];
-  });
+  const panels = useSelector((state) => state.local.popovers.activeNodeMetaPanels);
+  const activeNodeMetaPanels = [...Object.keys(panels)];
```
Comment (ylvaselling) [inline src/components/NodeMetaPanel/NodeMetaContainer.jsx:10]: 👍

File: src/components/Sidebar/Properties/PropertyOwnerHeader.jsx [hunk 2]
```diff
@@ -284,13 +291,13 @@ function PropertyOwnerHeader({
         </span>
         <span className={styles.rightButtonContainer}>
           { isSceneObject && focusButton }
-          { hasMoreButtons && (
+          { (showPopOutSettings || showMeta || showMeta) && (
```
Comment (ylvaselling) [inline src/components/Sidebar/Properties/PropertyOwnerHeader.jsx:294]:
```suggestion
          { (showPopOutSettings || showMeta) && (
```

**Review by ylvaselling — APPROVED** [review]
👍

File: src/components/Sidebar/Properties/PropertyOwnerHeader.jsx (hunk 2 above)
Comment (WeirdRubberDuck) [reply src/components/Sidebar/Properties/PropertyOwnerHeader.jsx:294]: Ah :) Actually, it should be: 
```suggestion
          { (showPopOutSettings || showMeta || trashAction) && (
```

**Pull request merged**